app.json

```diff
@@ -0,0 +1,24 @@
+{
+  "addons": [
```

Comment:
@walimike , please squash your commits. you have 22 commits on one PR. Please squash them.